src/editor.js

```diff
@@ -54,8 +54,9 @@ export const BACKSPACE_FIRST_COMMANDS = BACKSPACE_ONLY_COMMANDS.concat(['oEnter'
 
 const KEYBOARD_TYPES = { VIRTUAL: 'VIRTUAL', PHYSICAL: 'PHYSICAL', UNKNOWN: 'UKNOWN' };
 
-const isUndo = ev => ev.key === 'z' && (ev.ctrlKey || ev.metaKey);
-const isRedo = ev => ev.key === 'y' && (ev.ctrlKey || ev.metaKey);
+const IS_UNDO = ev => ev.key === 'z' && (ev.ctrlKey || ev.metaKey);
+const IS_REDO = ev => ev.key === 'y' && (ev.ctrlKey || ev.metaKey);
+const IS_BOLD = ev => ev.key === 'b' && (ev.ctrlKey || ev.metaKey);
```

Comment:
Not a big fan of the names (nor the ones before that I chose myself, I know). Maybe `IS_KEYBOARD_EVENT_SOMETHING`? Or not, if it adds too many LOCs.